Sources/KindeSDK/Auth/Auth.swift

```diff
@@ -93,6 +211,28 @@ public final class Auth {
         return nil
     }
     
+    public func getPermission(name: String, options: ApiOptions? = nil) async throws -> Permission {
+        if options?.forceApi == true {
+            let perms = try await getPermissions(options: options)
+            return Permission(organization: perms.organization, isGranted: perms.permissions.contains(name))
+        } else {
+            if let permissionsClaim = getClaim(forKey: ClaimKey.permissions.rawValue),
+               let permissionsArray = permissionsClaim.value as? [String],
+               let orgCodeClaim = getClaim(forKey: ClaimKey.organisationCode.rawValue),
+               let orgCode = orgCodeClaim.value as? String {
+                
+                let organization = Organization(code: orgCode)
+                let permission = Permission(organization: organization,
+                                            isGranted: permissionsArray.contains(name))
+                return permission
+            }
```

Comment:
Since the same logic is in the `getPermission(name: String)` function, we can use that directly here. 

```suggestion
        if let permission = getPermission(name: name) {
            return permission  
        }
```